common/djangoapps/terrain/stubs/youtube.py

```diff
@@ -119,7 +120,10 @@ def _send_video_response(self, youtube_id, message):
         callback = self.get_params['callback']
         youtube_metadata = json.loads(
             requests.get(
-                "http://gdata.youtube.com/feeds/api/videos/{id}?v=2&alt=jsonc".format(id=youtube_id)
+                "https://www.googleapis.com/youtube/v3/videos/?id={id}&part=contentDetails&key={key}".format(
+                    id=youtube_id,
+                    key=settings.XBLOCK_SETTINGS['VideoModule']['YOUTUBE_API_KEY']
+                )
```

Comment:
This makes the tests depend on the presence of the API key. I think we should just return some dummy value here instead.